ecs-cli/integ/e2e/local_test.go

```diff
@@ -0,0 +1,90 @@
+// +build integ
+
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/integ"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/integ/stdout"
+	"github.com/stretchr/testify/require"
+)
+
+func TestECSLocal(t *testing.T) {
+	t.Parallel()
+
+	type commandTest struct {
+		args    []string
+		execute func(t *testing.T, args []string)
+	}
+
+	tests := map[string]struct {
+		sequence []commandTest
+	}{
+		"clean state": {
```

Comment:
Is there a reason why this is a table driven test? 

Looks like it only has one test case...

Reply:
soon™ we'll have full ecs-cli local workflows as integration tests :P.

This sequence for example will make up on test case: "ecs-cli local create --arn" -> "ecs-cli local up" -> "ecs-cli local ps" -> "ecs-cli local stop"

Reply:
😎